### ISSUE TITLE
Support for AppleScript

### ISSUE THREAD

**Issue by fabiospampinato** [zeit/hyper#3410]
Under macOS one can execute the following applescript:

```applescript
tell application "Terminal" to do script "echo hello"
```

To open another Terminal window with the provided command.

It would be great if Hyper had similar scriptability capabilities.

**Comment by c-bandy** [comment]
Hyper is a terminal emulator, a shell process. How do you suppose it would support AppleScript? Correct me if I'm wrong but I don't believe the regular Mac terminal supports this functionality in any way.

**Comment by j-f1** [comment]
Upstream issue: https://github.com/electron/electron/issues/4418

**Comment by deanishe** [comment]
> Hyper is a terminal emulator, **a shell process**

No, it isn't. It's a GUI application that runs a shell process within it. Supporting AppleScript is very common amongst Mac apps aimed at power users and/or developers.

> Correct me if I'm wrong but I don't believe the regular Mac terminal supports this functionality in any way.

Completely wrong. @fabiospampinato literally posted example code for doing exactly this with the regular Mac terminal. iTerm2.app also has very good support for scripting via AppleScript.

**Comment by Triggs9** [comment]
do shell script "open -a Hyper /usr/local/microsoft/powershell/7-lts/pwsh"

it doesn't have to directly support Hyper, this command in an automator script will open hyper and run pwsh
